HoloPipelines/pipelineServer.py

```diff
@@ -0,0 +1,202 @@
+from flask import Flask, request
```

Comment:
This file does way too much stuff which is not server related. Please do at least extract other components:
* to keep the status of jobs
* to do the downloading and unzipping stuff